rockcraft/plugins/python_plugin.py

```diff
@@ -17,13 +17,25 @@
 """The Rockcraft Python plugin."""
 
 import logging
-from typing import Optional
+from textwrap import dedent
+from typing import List, Optional
 
 from craft_parts.plugins import python_plugin
 from overrides import override
 
 logger = logging.getLogger(__name__)
 
+# Template for the sitecustomize module that we'll add to the payload so that
+# the pip-installed packages are found regardless of how the interpreter is
+# called.
+SITECUSTOMIZE_TEMPLATE = dedent(
+    """
+    import site
+    site.addsitedir("/lib/${py_version}/site-packages")
```

Comment:
an alternative to the template would be to import sys here and use something like `sys.version_info.minor`
```suggestion
    import sys
    site.addsitedir(f"/lib/{sys.version_info.major}.{sys_version.minor}/site-packages")
```